### PR TITLE
Disallow answers less than 1 unless arg -g is passed in

### PR DESCRIPTION
**Pull request by eShuttleworth**
Probably not the best argument name, feel free to propose a change. 

In response to issue #67

### CODE REVIEW

**Comment by jcandan** [comment]
Good job! Right, and this is partially why I disincluded an argument from this issue. There's potential to allow user responses to a certain precision. I want to think through this a bit, and whether this feature is even necessary for the intended audience. Please remove the argument, and simply always ensure non-decimal answers are formed. Thanks for the PR!

**Comment by jcandan** [comment]
There's even the option to consider simplifying fractions as opposed to decimal values, which might end up being the route I take. I plan to talk with some elementary teachers before making that call first.

**Comment by eShuttleworth** [comment]
Hmm, I just finished a feature to round to 3 decimal places, I'd imagine that would be preferable. After all, it is only division that is affected. Another option could be whole number+remainder.

**Comment by jcandan** [comment]
yes, whole number + remainder is another option. thanks for that too! which, again, is why i will forgo this as optional for now.

**Comment by jcandan** [comment]
Did you want to complete the removal of the optional logic so we can merge this in?

**Comment by eShuttleworth** [comment]
Sorry it took me a few, arg removed.

File: math_quiz.py [hunk 1]
```diff
@@ -99,4 +102,4 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+	main()
```
Comment (jcandan) [inline math_quiz.py]: Overindented. Could you fix?

File: math_quiz.py [hunk 2]
```diff
@@ -46,6 +46,7 @@ def build_questions():
     parser.add_argument("-m", "--multiply", help = "Multiply the terms", action = "store_true")
     parser.add_argument("-d", "--divide", help = "Divide the terms", action = "store_true")
     parser.add_argument("-n", "--negative", help = "Allow negative equations", action = "store_true")
+
```
Comment (jcandan) [inline math_quiz.py]: Not a big deal, but got an extra line here for no reason. please remove?

**Review by jcandan — COMMENTED** [review]
Just a couple of minor items to fix. and you'll have a merged PR! thanks again for tackling this!

File: math_quiz.py (hunk 1 above)
Comment (eShuttleworth) [reply math_quiz.py]: Fixed, not sure how that happened

**Comment by jcandan** [comment]
okay, I see we're going to have to rethink how we tackle this. 

```
$ python math_quiz.py -dr
4 / 4 = 1
Incorrect :( . . . The answer is 1.0
7 / 3 = 2.25
Incorrect :( . . . The answer is 2.3333333333333335
```

Your PR exactly met the requirements as stated in #67, but that did little to actually stop decimal values from being used, which is completely my mistake. I will close #67 and open a new issue with a more effective, and actually more relevant solution.

**Pull request closed without merging**